code/modules/antagonists/changeling/changeling.dm

```diff
@@ -714,6 +717,9 @@
 	user.grad_style = LAZYLISTDUPLICATE(chosen_profile.grad_style)
 	user.grad_color = LAZYLISTDUPLICATE(chosen_profile.grad_color)
 	user.physique = chosen_profile.physique
+	user.selected_scream = new chosen_profile.scream_type
```

Comment:
are the lingering scream/laugh references deleted anywhere else?

Reply:
Uh-oh, no I don't believe so. Do you mean that when a new scream/laugh is made for the character, the old one doesn't get removed?

Would it make sense to select and delete the current one before making a new one? If so, how would I go about that?

Reply:
I just used qdel(), let me know if that's not enough.